Formula/php-solr.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "../lib/php_pecl_formula"
+
+class PhpSolr < PhpPeclFormula
+  extension_dsl "PHP extension for Apache Solr"
```

Comment:
Please use the (slightly edited) summary from the pecl package: `An extremely fast, light-weight, feature-rich library that allows PHP applications to communicate easily and efficiently with Apache Solr server instances using an object-oriented API`

Reply:
didn't brew reject that long summary? i.e limiting to something very small like 40 characters?

**why** do you want that long description? what purpose does it serve? it's not like it's used for search keywords? if I want to read more about the extension, I can visit the homepage, or use search engine to find blog posts.

I personally dislike such promotive description - that is useless to me as a developer, yada-yada extremely fast, lightweight, efficient - it's very opinionated. what matters is that its Apache Solr client, that provides OOP classes.

Reply:
what I found useful is to specify solr server version:

```
➔ brew info php@7.3-solr
kabel/pecl/php@7.3-solr: stable 2.5.1 [keg-only]
PHP extension for communicating with Apache Solr Server 4.0+ for PHP 7.3
https://pecl.php.net/package/solr
Not installed
From: https://github.com/kabel/homebrew-pecl/blob/HEAD/Formula/php@7.3-solr.rb
License: PHP-3.01
==> Dependencies
Build: autoconf ✔, pkg-config ✔
Required: php@7.3 ✔, curl-openssl ✔
==> Caveats
php@7.3-solr is keg-only, which means it was not symlinked into /usr/local,
because this is an alternate version of another formula.

```

took this from https://www.php.net/manual/en/intro.solr.php

Reply:
I tend to agree with you about the overly long, promotional, etc. descriptions, but I'm trying to follow a direct mapping to the pecl packages registry and this is what the published summary is. Example from terminal:

```
$ pecl search solr
Matched packages, channel pecl.php.net:
=======================================
Package Stable/(Latest) Local
solr    2.5.1 (stable)        The Apache Solr PHP extension is an extremely fast,
                                  light-weight, feature-rich library that allows PHP applications to
                                  communicate easily and efficiently with Apache Solr server instances
                                  using an object-oriented API.
```

It's not up to me to editorialize the extension authors.

Reply:
can this be merged now?

```
[/usr/local/Homebrew/Library/Taps/kabel/homebrew-pecl (pecl-solr)↑★] ➔ brew info php-solr
kabel/pecl/php-solr: stable 2.5.1
The Apache Solr PHP extension is an extremely fast, light-weight, feature-rich library that allows PHP applications to communicate easily and efficiently with Apache Solr server instances using an object-oriented API for PHP 7.4
https://pecl.php.net/package/solr
Not installed
From: https://github.com/kabel/homebrew-pecl/blob/HEAD/Formula/php-solr.rb
License: PHP-3.01
==> Dependencies
Build: autoconf ✔, pkg-config ✔
Required: php ✔, curl-openssl ✔
[/usr/local/Homebrew/Library/Taps/kabel/homebrew-pecl (pecl-solr)↑★] ➔
```